packages/legacy-scripting-runner/index.js

```diff
@@ -207,9 +212,44 @@ const parseFinalResult = async (result, event) => {
   return result;
 };
 
+const serializeValueForCurlies = value => {
+  if (typeof value === 'boolean') {
+    return value ? 'True' : 'False';
```

Comment:
I might do these as lowercase. Who knows how the external server reads them, but since that's how they appear in JSON, lowercase feels safer to me.

Reply:
This is actually how WB behaves. If you write `https://example.com?active={{is_active}}` for a trigger/action URL and `is_active` is a boolean, the boolean going to be serialized to `True` or `False`.